src/components/FlightsTable/FlightsTable.jsx

```diff
@@ -7,9 +7,9 @@ export const FlightsTable = ({originFilter, flights}) => {
     <section className="flightsTable">
       <h3>Flights</h3>
```

Comment:
Semantically speaking, without an `<h2>`, let alone an `<h1>`, you shouldn't have an `<h3>`. Header hierarchy must be exact, you cannot jump levels.

Reply:
Did not know that, but makes sense. Fixed.

Reply:
@netia1128 Typically you would have the `<h1>` in some sort of site header -- usually the name of the site ("SkyFlow") -- and then all of your main level headings would be `<h2>` (typically associated with the first set of `<article>` or `<section>`) and then subheadings (individual content blocks within the article or sections) as `<h3>` etc.

Reply:
This is helpful. If you ever happen to stumble across any articles or websites you think do a good job of explaining semantic tag nesting, I'd love to wrap my head around the basics if nothing else.

Reply:
Here's the specifics: https://www.a11yproject.com/posts/how-to-accessible-heading-structure/

You can start here for essential semantics in general: https://www.a11yproject.com/checklist/